### ISSUE TITLE
[BB] Bilerot Sickness debuff during boss fights

### ISSUE THREAD

**Issue by Vayra** [WarEmu/WarBugs#16621]
**Expected behavior and actual behavior:**
The Bilerot Sickness debuff (ap drain) should not be active during boss fights. Since last patch, it happens during boss fights as well as when running there. So far it's happened on Ssrydian Morbidae. This fight is already a tough gear check for groups that aren't overgeared already and this bug makes the fight far harder than it should be.

**Steps to reproduce the problem:**
Fight the boss and the debuff is put on you during the fight. 

**Testing Screenshots/Videos/Evidences (always needed):**

![Kalyth_009](https://user-images.githubusercontent.com/1183293/80846218-d4be6d80-8c0b-11ea-9a05-5c440fbb8f16.jpg)

**Comment by Kaelang** [comment]
Was you using Gag Self to remove the debuff?

**Comment by Vayra** [comment]
yes, but that's not the point, the debuff should not be applied during the boss fight. This is already a dps check, and getting an AP drain on the boss makes it significantly harder unless you already outgear it by a few tiers.

**Comment by Vayra** [comment]
Another issue with the debuff is that sometimes it seems you can have 2 stacks of it, so that even if you puke, you still keep the debuff.

**Comment by Kaelang** [comment]
In terms of the debuff in itself, it's intended - the tooltip just needs updating.

'It’s intended , I need to update the tooltip probably . The granted abiltiy last few sec , if you don’t cleanse yourself quick enough, the debuff will stay'

In regards to the DoT being applied during the boss fight, it's highly likely that it's overran from a previous pack. I'll wait for confirmation on this.

**Comment by Vayra** [comment]
It applies during the bossfight, none of us had it when we started it. And multiple people would get it throughout the fight. Here's a screenshot of 2 of us having it while the boss is at 50%. You can see I still have the Gag Self ability here since I just got it at that point. 
![Kalyth_010](https://user-images.githubusercontent.com/1183293/80863230-20f4c680-8c7b-11ea-9743-9ff3581c76d7.jpg)

**Comment by grunbag** [comment]
You need to use gag self as soon as it get up , otherwise after few sec you will be unable to cleanse the AP drain

**Comment by grunbag** [comment]
The onscreen text is telling you what to do to deal with other debuff and dot

**Comment by Vayra** [comment]
I don't think you realize what the report is about Grunbag. Prior to last patch, this boss did NOT apply the Bilerot Sickness debuff during the fight. Now he does. There is no patch notes to suggest that should be the case. And the boss did not use this debuff on live or any prior patch. I know full well how to deal with the debuff, the point is it should not be happening during the boss fight.

**Comment by grunbag** [comment]
The boss always added the debuff it’s part of its mechanics since day 1

**Comment by grunbag** [comment]
And the boss used this debuff on live too check the prima guide

**Comment by Vayra** [comment]
Not the AP drain, I have never gotten that debuff during this boss fight either here or on live before last night.

**Comment by Vayra** [comment]
The boss has had 3 debuffs, one for each cauldron. And the Rotting Zymosis to act as a timer for the boss. Never the AP drain.

**Comment by grunbag** [comment]
CHECk vous 2m30 you can see the dok having the gag self

https://youtu.be/l_p3jbMG0sc

**Comment by Vayra** [comment]
Interesting, but as you can see in that video, it's not an AP drain, it's a reduced regen. The DoK never loses AP while he has the debuff, and can cleanse it at any time. So either way there's a bug here. 

And remember, this dungeon gives Conq level gear, it should not require higher gear than that to beat. With the AP drain active, a group of people geared in Anni/Redeye has 0 chance against this boss.

**Comment by grunbag** [comment]
As I said in an old ticket about gag self , we changed the AP regen réduction to à AP drain . It a ror design since day one . You maybe haven’t noticed it until today or haven’t receive d the gag self dunno but nothing changed for dungeon mechanics since a long time

**Comment by grunbag** [comment]
You can easily remove the AP drain by clicking on gag self as soon as it get up

**Comment by Vayra** [comment]
Clearly something changed with the patch since over the past however many runs I've done of BB I've never gotten it and neither has anyone else in my party. 

And while we can remove the drain, that spawns an add that takes away damage from the boss and makes tanking harder. Which isn't an issue for a well-geared group, but if you're at the intended gear level, this debuff takes the fight from hard but doable to impossible.

**Comment by grunbag** [comment]
It has only 50% chance to spawn an add if I recall , and the add is fairly easy to kill . We haven’t change anything , I’m 100% see as I designed all dungeon boss mechanics and I am in charge of abilities . I haven’t worked on dungeon since 2019

**Comment by Vayra** [comment]
I'll check if it's 50%, it feels guaranteed, but haven't paid too much attention to that. But something definitely changed, we ran our mains through on Tuesday and no debuff. Then last night we did an alt run and got completely taken aback by the debuff suddenly being used. Maybe it's been bugged and not using it for the past few months, I don't know, but something changed with the patch and it's making it ridiculously hard for groups that don't significantly outgear the dungeon.
